vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -7,117 +7,175 @@
-pods expecting that it will apply the recommendation when the pod is recreated. This is a
-disruptive process so VPA has some mechanism to avoid too frequent disruptive updates. 
+pods expecting that it will apply the recommendation when the pod is recreated. Today, this process
+is potentially disruptive as any changes in recommendations requires a pod to be recreated.
+
+We can instead reduce the amount of disruption by leveraging the [in-place update feature] which is
+currently an [alpha feature since 1.27] and graduating to [beta in 1.33].
 
-This proposal allows VPA to apply its recommendations more frequently, with less disruption by
-using the
-[in-place update feature] which is an alpha feature [available in Kubernetes 1.27.] This proposal enables only core uses
-of in place updates in VPA with intention of gathering more feedback. Any more advanced uses of in place updates in VPA
-(like applying different recommendations during pod initialization) will be introduced as separate enhancement
-proposals.
+This proposal enables only core uses of in place updates in VPA with intention of providing the
+foundational pieces. Further advanced uses of in place updates in VPA (like applying different
+recommendations during pod initialization or providing more frequent smaller updates) will be
+introduced as separate enhancement proposals.
 
 [in-place update feature]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources
-[available in Kubernetes 1.27.]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#api-change-3
+[alpha feature since 1.27]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#api-change-3
+[beta in 1.33]: https://github.com/orgs/kubernetes/projects/178/views/1
+
+### A Note On Disruptions {#disruptions}
+
+It is important to note that **VPA cannot guarantee NO disruptions**. This is because the
+underlying container runtime is responsible for actuating the resize operation and there are no
+guarantees provided (see [this thread] for more information). However, in practice if the
+underlying container runtime supports it, we expect these disruptions to be minimal and that MOST
+of the time the updates will be done in-place.
+
+This proposal therefore focuses on *reducing* disruptions while still harnessing the benefits of
+VPA.
+
+[this thread]: https://github.com/kubernetes/autoscaler/issues/7722#issue-2796215055
 
 ### Goals
 
-* Allow VPA to actuate without disruption,
-* Allow VPA to actuate more frequently, when it can actuate without disruption,
-* Allow VPA to actuate in situations where actuation by eviction doesn't work.
+* Allow VPA to actuate with reduced disruption.
+* Allow VPA to actuate in situations where actuation by eviction is not desirable.
 
 ### Non-Goals
 
+* Allow VPA to actuate more frequently.
+* Allow VPA to operate with NO disruptions, see the [note above](#disruptions).
```

Comment:
```suggestion
* Allow VPA to operate with NO disruptions, see the [note above](#a-note-on-disruptions-disruptions).
```

I think that will work?

There are a few of these anchor links that are broken currently

Reply:
I think I've fixed them all. Thanks for finding this.